samples/apps/copilot-chat-app/webapi/CopilotChat/Skills/ChatSkills/DocumentMemorySkill.cs

```diff
@@ -47,7 +47,7 @@ public class DocumentMemorySkill
     /// </summary>
     /// <param name="query">Query to match.</param>
     /// <param name="context">The SkContext.</param>
```

Comment:
Update /// <params> to match args please :)

Reply:
(I'm also fine just deleting args and keeping only the <summary> for the non-SDK code, as long that doesn't throw warnings)